### ISSUE TITLE
[KEYBOARD]: Facility links are missing the yellow focus halo

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#2275]
## Description

While testing another issue, I noticed our list of facility links don't have the yellow focus halo when I tab to them. I've verified this with `document.activeElement` and attached a screenshot below showing the item to correct.

## Point of Contact

**VFS Point of Contact:** _Trevor_

## Acceptance Criteria
[ ] Keyboard user can see the yellow focus halo when the link **for any facility type** has focus.
[ ] Run automated scans periodically during development
[ ] Internal peer review 
[ ] After peer review, share with the FL team for Product/Design sign off and/or FE Review Team 
[ ]  Documentation and points of contact for any new backend dependencies
[ ]  Links to important dashboards for investigating relevant issues

## Definition of done
[ ] Events are logged appropriately
[ ] Documentation has been updated, if applicable
[ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
[ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

## WCAG or Vendor Guidance (optional)

* [Focus Visible: Understanding SC 2.4.7](https://www.w3.org/TR/UNDERSTANDING-WCAG20/navigation-mechanisms-focus-visible.html)

## Screenshots or Trace Logs

![Screen Shot 2019-10-04 at 4 58 40 PM](https://user-images.githubusercontent.com/934879/66243156-1f552700-e6c9-11e9-8222-18e11faae52f.png)

**Comment by Karak888** [comment]
thanks @1Copenut much appreciated!

**Comment by mmiddaugh** [comment]
@brangi this one can be evaluated for level of effort and user impact.

**Comment by brangi** [comment]
@1Copenut 
<img width="881" alt="Screen Shot 2019-12-19 at 8 27 33 PM" src="https://user-images.githubusercontent.com/100468/71225374-11617c00-229e-11ea-8a10-6ef442e47d1c.png">

Is this the one ^ ? because it looks like it's there

**Comment by 1Copenut** [comment]
@brangi Good morning. I spot-checked this on Staging quickly to see if the focus state was being picked up, but didn't see the yellow halo when I navigated using the `TAB` key. I verified the current element under focus in the screenshot below. Let's keep this issue open and treat as unresolved.

---
![Screen Shot 2019-12-20 at 8 32 08 AM](https://user-images.githubusercontent.com/934879/71261783-e01b9700-2303-11ea-82f2-a88601b398ed.png)

**Comment by mmiddaugh** [comment]
Closing as unresolved/unable to reproduce
